thorlcr/activities/fetch/thfetch.cpp

```diff
@@ -111,9 +111,16 @@ class CFetchActivityMaster : public CMasterActivity
         for (unsigned i=0; i<numFilesToRead; i++)
             fileStats[i]->deserialize(node, mb);
     }
+    virtual void getActivityStats(IStatisticGatherer & stats) override
+    {
+        CMasterActivity::getActivityStats(stats);
+        diskAccessCost = calcFileReadCostStats(false);
+        if (diskAccessCost)
+            stats.addStatistic(StCostFileAccess, diskAccessCost);
```

Comment:
future: instead of aggregating storing (diskAccessCost) as a member, and aggregating in CMasterGraph::getDiskAccessCost(), could the StatisticsAggregator which currently stops at the sg level, be responsible for aggregating these up, so done in a generalized way instead?

Reply:
It could be.  I think there are other activity stats that can be aggregated by StatisticsAggregator.

Reply:
@shamser - can you open a cost 2.0 subtask to remember to revisit this?